pyrk/RELEASE-VERSION

```diff
@@ -0,0 +1 @@
+0.1.dev44a238e
```

Comment:
This RELEASE-VERSION file should be left out of the PR. It looks like it's in the gitignore, and I see you tried to fix the way it's ignored. But, it seems like you added it before this took effect. To avoid accidentally adding files like this, try to never type "git add *" . Explicitly add the few files you want by name, when you want to add new files to version control.